portal/src/walrus-sites-sw.ts

```diff
@@ -6,7 +6,9 @@ import * as baseX from "base-x";
 import { fromB64, fromHEX, isValidSuiObjectId, isValidSuiAddress, toHEX } from "@mysten/sui/utils";
 import { AGGREGATOR, SITE_PACKAGE, SITE_NAMES, NETWORK, MAX_REDIRECT_DEPTH } from "./constants";
 import { bcs, BcsType } from "@mysten/bcs";
-import template_404 from "../static/404-page.template.html";
+import template_404 from "@static/404-page.template.html";
```

Comment:
It's a convention of naming paths in `tsconfig.json`. 
[TS path definitions](https://www.typescriptlang.org/tsconfig/#paths) are a way of mapping relative paths to predetermined ones. i.e. `"@static": "../static"`. 
I added it so that it looks nicer, no other reason.

Reply:
thanks for the explanation!